iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    public class CleanupDevicesSample
```

Comment:
Whole class could use more commenting as to approach.

Reply:
I've added a comment for each step taken. Is there some other information that you'd like to see here? (maybe about specific access permissions granted to the sas token?)

Reply:
Looks good.